src/main/java/use_cases/SavePlayer.java

```diff
@@ -0,0 +1,35 @@
+package use_cases;
```

Comment:
I'm not too sure about including this in the use_cases packages, it is too concerned with the way we are saving Players (the "how"). I'd go with Edric's suggestion here.

Reply:
I had to include the use_cases package here because SavePlayer is in the use_cases package.